src/v2/views/ErrorView/index.tsx

```diff
@@ -12,6 +12,8 @@ import bytes from "bytes";
 import { getBlockChainStorePath, userConfigStore, app } from "src/config";
 
 const transifexTags = "v2/ErrorView";
+const downloadLink =
+  "https://release.nine-chronicles.com/NineChroniclesInstaller.exe";
```

Comment:
You can use `baseUrl` instead `https://release.nine-chronicles.com`
https://github.com/planetarium/9c-launcher/blob/cdf051eca964581c9e034fa65e33df559443e5b3/src/config.ts#L224 

and recommend define `var installerName = "NineChroniclesInstaller.exe"` in config.ts

Reply:
Oh thx! 👍🏻 
I fix it